src/app/complex/complex-results/complex-results.component.css

```diff
@@ -51,30 +51,42 @@
   animation: animateOutline 4s ease infinite;
 }
 
+.cnButton.selected {
+  animation: none;
+}
+
 @keyframes animateOutline {
-  0% {
-    outline-width: 1px;
-    outline-offset: 0;
+  0%,
+  50%,
+  100% {
+    outline-width: 6px;
+    outline-offset: 4px;
     outline-color: rgba(0, 124, 130, 0);
   }
-
   10% {
     outline-color: rgba(0, 124, 130, 0.75);
   }
+}
 
-  50% {
-    outline-width: 6px;
-    outline-offset: 4px;
-    outline-color: rgba(0, 124, 130, 0);
-  }
+.smallCN {
+  width: 55vw;
+  float: right;
+  margin-left: auto;
+}
 
-  100% {
-    outline-width: 6px;
-    outline-offset: 4px;
-    outline-color: rgba(102, 102, 102, 0);
-  }
+.changingOfDisplay {
+  float: left;
+  width: 75vw;
+  margin-right: -2vw;
 }
 
-.cnButton.selected {
-  animation: none;
+.column:last-child:not(:first-child),
+.columns:last-child:not(:first-child) {
+  float: left !important;
+}
+
+.listOfResults {
+  width: 40vw;
+  margin-left: 17vw;
```

Comment:
If you change this to 10vw, it may look a bit better `margin-left: 17vw;`

Reply:
After this and the other CSS change mentioned above, try in Chrome, Safari and Firefox to check everything looks ok.